src/main/java/org/spideruci/tacoco/MavenBuildProbe.java

```diff
@@ -103,14 +105,23 @@ public BuilderType getBuilderType() {
 	public String getClasspath(){
 		try{
 			if(classpath != null) return classpath;
-			if(!new File(targetDir+"/tacoco.cp").exists()) {
+            final String tacocoCpPath = new PathBuilder().path(targetDir).path("tacoco.cp").buildFilePath();
+
+			if(!new File(tacocoCpPath).exists()) {
                 MavenCli mavenCli = new MavenCli();
-                mavenCli.doMain(new String[] {"dependency:build-classpath", "-Dmdep.outputFile=tacoco.cp"}, targetDir,
+                mavenCli.doMain(new String[]{"dependency:build-classpath", "-Dmdep.outputFile=tacoco.cp"}, targetDir,
                         System.out, System.out);
 			}
-			classpath = new String(Files.readAllBytes(Paths.get(targetDir,"tacoco.cp")))
-					+":"+ targetDir + "/target/test-classes"
-					+":"+ targetDir + "/target/classes";
+
+            final String tacocoDependencies = new String(Files.readAllBytes(Paths.get(targetDir, "tacoco.cp")));
+            final String targetPath = new PathBuilder().path(targetDir).path("classes").buildFilePath();
```

Comment:
How was this even working in the first place?? there should have been a `.path("target")` in this call chain -- and in the call chain in the next line. It was a small error and i fixed it. But i was wondering if it worked for you (if you tested it before sending the pull request)?